app/tests/integration/nimbus/test_firefox_targeting.py

```diff
@@ -133,3 +133,82 @@ def test_check_targeting(
             context="chrome",
         )
     assert result is not None, "Invalid Targeting, or bad recipe"
+
+
+@pytest.mark.parametrize(
+    "audience_field",
+    [
+        {"channel": "NIGHTLY"},
+        {"firefoxMinVersion": "FIREFOX_100"},
+        {"firefoxMaxVersion": "FIREFOX_120"},
+        {"locales": [37]},
+        {"countries": [42]},
+        {"proposedEnrollment": "14"},
+        {"proposedDuration": "30"},
+    ],
+    ids=[
+        "channel",
+        "min_version",
+        "max_version",
+        "locales",
+        "countries",
+        "proposed_enrollment",
+        "proposed_duration",
+    ],
+)
+@pytest.mark.run_targeting
+def test_check_audience_targeting(
+    selenium, slugify, experiment_name, create_desktop_experiment, audience_field
+):
+    experiment_slug = str(slugify(experiment_name))
+    data = {
+        "hypothesis": "Test Hypothesis",
+        "application": "DESKTOP",
+        "changelogMessage": "test updates",
+        "targetingConfigSlug": "no_targeting",
+        "publicDescription": "Some sort of Fancy Words",
+        "riskRevenue": False,
+        "riskPartnerRelated": False,
+        "riskBrand": False,
+        "featureConfigId": 1,
+        "referenceBranch": {
+            "description": "reference branch",
+            "name": "Branch 1",
+            "ratio": 50,
+            "featureEnabled": True,
+            "featureValue": "{}",
+        },
+        "treatmentBranches": [
+            {
+                "description": "treatment branch",
+                "name": "Branch 2",
+                "ratio": 50,
+                "featureEnabled": False,
+                "featureValue": "",
+            }
+        ],
+        "populationPercent": "100",
+        "totalEnrolledClients": 55,
+    }
```

Comment:
We don't have like a fixture for this in conftest somewhere?

Reply:
Good idea, we can add that